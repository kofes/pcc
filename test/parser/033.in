var
	arr : array[0..
