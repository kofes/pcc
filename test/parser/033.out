Illegal expression: 'MOD' in pos (1, 7);
